ex3/server/StandardIO.h

```diff
@@ -0,0 +1,16 @@
+#include "DefaultIO.h"
+
+#ifndef CLIENT_STANDARDIO_H
+#define CLIENT_STANDARDIO_H
+
+
+class StandardIO : public DefaultIO {
+public:
+    void write(std::string string) override;
+
+    std::string read() override;
+
```

Comment:
```suggestion
```

Reply:
This is a very bad change tom, it will make the whole project very very bad